Gemfile.lock

```diff
@@ -1,17 +1,22 @@
+PATH
```

Comment:
Convention is to drop the Gemfile.lock if you have a gemspec file, I think you can remove this file (:

Reply:
I think it made it automatically when I did bundle install; do you mean that I should gitignore it?